paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/common/DropdownElement.kt

```diff
@@ -4,15 +4,21 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.map
 
+/**
+ * This class controls the dropdown view and implements the [Element] interface.
+ * Because it can never be in error the `errorMessage` is always null.  It is also
+ * designed to always have a value selected, so isComplete is always true.
+ */
 internal class DropdownElement(
-    private val config: DropdownConfig,
-) {
+    config: DropdownConfig,
+) : Element {
     val displayItems: List<String> = config.getDisplayItems()
-
     private val _selectedIndex = MutableStateFlow(0)
     val selectedIndex: Flow<Int> = _selectedIndex
-
-    val paymentMethodParams = selectedIndex.map { config.getPaymentMethodParams()[it] }
+    override val label: Int = config.label
+    override val fieldValue = selectedIndex.map { displayItems[it] }
+    override val errorMessage: Flow<Int?> = MutableStateFlow(null)
+    override val isComplete: Flow<Boolean> = MutableStateFlow(true)
```

Comment:
Probably worth sharing that there's also a `MutableSharedFlow` that is basically a `MutableStateFlow` without a value. Any observer collecting on a `MutableSharedFlow` will receive its emission.

This is useful in some scenarios where `value` isn't needed.